indigo/distribution.yaml

```diff
@@ -8399,6 +8399,23 @@ repositories:
       type: git
       url: https://github.com/silviomaeta/plot_util.git
       version: master
+  plotjuggler-release:
```

Comment:
Same comment as for the previous PR: https://github.com/ros/rosdistro/pull/13670#discussion_r97373216